docs/guides/cli.md

```diff
@@ -52,6 +53,71 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+```bash
```

Comment:
I believe it should be a couple of lines below.

Reply:
You are right :)